gramex/handlers/mlhandler.py

```diff
@@ -112,14 +114,26 @@ def setup(cls, data=None, model={}, config_dir='', **kwargs):
             data = cls._filtercols(data)
             data = cls._filterrows(data)
             cls.model = cls._assemble_pipeline(data, mclass=mclass, params=params)
-
             # train the model
             target = data[target_col]
             train = data[[c for c in data if c != target_col]]
+            # cross validation
+            mod = cls.modelFunction()
+            CVscore = cross_val_score(mod, train, target)    
+            CV = sum(CVscore)/len(CVscore)
```

Comment:
1. Use `CVscore.mean()`
2. Variable naming has to follow a specified style - do `pip install flake8` and run the `flake8` command against this file, i.e. `flake8 mlhandler.py`, and check the output.